src/options.js

```diff
@@ -2,6 +2,41 @@ import merge from 'deepmerge';
 import Handlebars from 'handlebars';
 import yaml from 'js-yaml';
 import marked from 'marked';
+import frontMatter from 'front-matter';
+
+var parsers = {
+  content: {
+    pattern: /\.(html|hbs|handlebars)$/,
+    parseFn: (contents, path) => {
```

Comment:
No, but parsers are invoked with a second argument (`path`). I left it in to make the code more self-documenting. Would you like me to pull it?

I can definitely see situations where you'd want `path` information to make decisions about parsing; that's why it's there :).